SKFlatMaker/src/SKFlatMaker.cc

```diff
@@ -2245,6 +2303,14 @@ el->deltaEtaSuperClusterTrackAtVtx() - el->superCluster()->eta() + el->superClus
       else{
         IDBit &= ~(1 << it_ID);
       }
+
+      //==== bits for each cuts
+      //==== https://twiki.cern.ch/twiki/bin/viewauth/CMS/SWGuideCMSPhysicsObjectSchoolAACHEN2019EGamma, Exercise 2
+      int this_idcutbid = 0;
+      if( el->hasUserInt(electron_IDtoSave.at(it_ID)) ){
+        this_idcutbid = el->userInt(electron_IDtoSave.at(it_ID));
+      }
+      electron_IDCutBit.push_back( this_idcutbid );
```

Comment:
If **hasUserInt** returns **false**, 0 is inserted to **electron_IDCutBit**. But, I think it possibly cause a mistake when we run analyzer. how about insert a value like -1 when **hasUserInt** returns **false**.